frontend/src/layouts/DashboardLayout.vue

```diff
@@ -177,7 +177,7 @@ export default {
           } = result
           this.GdtBalance =
             transactionList.balance.balanceGDT === null
-              ? null
+              ? 0
```

Comment:
Do we really want this? If for some reason the GDT server is not working. this may confuse the users a lot. `null` renders as `-`

Reply:
then we must test this times through what really happens if the GDT server times really does not go. I had yesterday the case that the GDT server did not go and **null** was detected as an error. 
I have but later noticed. so I have changed this to **0**. I change this back again.